castervoice/lib/ccr/html/html.py

```diff
@@ -1,8 +1,29 @@
 
 from castervoice.lib import control
-from castervoice.lib.actions import Key, Text
+from castervoice.lib.context import AppContext
+from castervoice.lib.actions import Key
 from castervoice.lib.dfplus.merge.mergerule import MergeRule
 from castervoice.lib.dfplus.state.short import R
+from castervoice.lib import settings
+
+# HTML auto complete workaround for Jetbrains Products
+# If more applications have issues with auto complete may be moved to actions.py
+context = AppContext(executable="idea", title="IntelliJ") \
+    | AppContext(executable="idea64", title="IntelliJ") \
+    | AppContext(executable="studio64") \
+    | AppContext(executable="pycharm")  \
+    | AppContext(executable="webstorm64") \
+    | AppContext(executable="webstorm")
```

Comment:
Can pass a list to `AppContext` as in something like `AppContext(executable=["studio64", "pycharm"]) ` etc

Reply:
Thanks that would be much cleaner approach.